orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -122,16 +124,25 @@ def _forward(self, *inputs, **kwargs):
                 self._get_inference_graph_and_init_gradient_graph_builder(
                     *inputs, **kwargs)
 
+            # Flag to indicate whether the gradient_graph needs to be built
+            build_gradient_graph = self._current_input_shape is None
             _, _, input_names_require_grad, new_input_shape = \
                 _ortmodule_io.parse_inputs_for_onnx_export(
                     self._original_module_parameters, self._onnx_inference, *inputs, **kwargs)
+            initializer_names_to_train_set = {name for name, param in
+                self._flattened_output_module.named_parameters() if param.requires_grad}
+            initializer_names_to_train_set_prev = set(self._onnx_graphs_info.initializer_names_to_train) \
```

Comment:
Regarding to the naming, initializer_names_to_train_set_prev actually refers to the current list of initializers (not prev) and  initializer_names_to_train_set refers to the new initializers?

I mean, using the onnx graph as ground truth, any property at self._onnx_graphs_info is the current value. Anything coming through forward is a possibly new thing

Reply:
renamed it as ```initializer_names_to_train_set_user_model``` and ```initializer_names_to_train_set_onnx_graph``` to avoid any confusion.